runtime/near-vm-runner/RUNTIMES.md

```diff
@@ -0,0 +1,176 @@
+This document describes the considerations that go into picking a Wasm runtime as well as the
+conclusions we made after evaluating a number of VM backends and frontends.
+
+# Criteria
+
+There are a number criteria which have been used to evaluate a Wasm runtime for use in a NEAR
+validator. Some of them are already listed in the
+[FAQ](FAQ.md#practical-aspects-of-the-execution-engines) document. Listed roughly in the order of
+importance:
+
+* correctness – does the runtime do what it claims to do;
+* security – how well does the runtime deal with untrusted input;
+* reliability – how much confidence is there in the implementation of runtime;
+* platform support – does the runtime support targets that we want to target;
+* runtime performance – how quickly can the runtime execute the prepared Wasm code;
+* preparation performance – how quickly can the runtime start running Wasm code;
```

Comment:
I consider `x86_64-linux` to be a part of such platform support requirement. It is not inconceivable that otherwise a very nice wasm compiler might only support targeting, say, [JVM bytecode](https://github.com/cretz/asmble), at which point it is most likely not applicable for our use cases, regardless of performance it can achieve.

Reply:
Yeah, we def need x86 support, support for everything else is niceto have.